ir/vector.h

```diff
@@ -67,16 +67,18 @@ class Vector : public VectorBase {
     explicit Vector(JSONLoader &json);
     Vector &operator=(const Vector &) = default;
     Vector &operator=(Vector &&) = default;
-    explicit Vector(const T *a) { vec.emplace_back(std::move(a)); }
+    explicit Vector(const T *a) { vec.emplace_back(a); }
```

Comment:
note that `move` on pointer has no effect

Reply:
Do we have similar things in `IndexedVector`?

Reply:
yep, we have :-/ Fixed